assets/js/editor-script-block.js

```diff
@@ -1,8 +1,9 @@
-/** 
+/**
  * Remove squared button style
- * 
+ *
  * @since 1.0.0
-*/
-wp.domReady( () => {
+ */
+/* eslint-disable no-undef */
```

Comment:
@ntwb, @aristath or @fabiankaegy: As I've seen the warning `'wp' is not defined.eslint(no-undef)` I've added `/* eslint-disable no-undef */`. Is this the correct way of handling it? I do have to say that I'm not very familiar with es-lint, that's why I reaching out to you.

Reply:
It's preferable if you just add this at the top of the file:
```
/* global wp, _ */
```
You can add others there too. In my example I'm defining wp and underscores, but you can add the ones you need comma-separated

Reply:
Great! Thanks for your reply, @aristath! Great to learn how to handle these issues properly on the go. 😀

Reply:
@aristath With https://github.com/WordPress/twentytwenty/pull/577/commits/ea36ed9c4a5542dcfb8c866007b7ed7bcc6566ab I brought this PR in proper shape. 😀